### ISSUE TITLE
Map markers don't work anymore

### ISSUE THREAD

**Issue by Gordin** [syyePhenomenol/HollowKnight.VanillaMapMod#4]
With the Mod active, I can buy infinite markers from the map shop, but the option in the Map screen to place markers is gone. It might be a problem with a save file, because I played Hollow Knight on PC with this Mod, and later played a bit on my steam deck without mods. I already tried to edit the save game, set all the `hasMarker...` variables to false, and bought the markers again from the shop, but it didn't fix my save, I can only place markers when the mod is disabled.

**Comment by syyePhenomenol** [comment]
Hi, would you be able to provide a ModLog.txt file?

```
"ModLog.txt" can be found in the same folder as your save files:
Windows: %APPDATA%\..\LocalLow\Team Cherry\Hollow Knight\
MacOS: ~/Library/Application Support/unity.Team Cherry.Hollow Knight/
Linux: ~/.config/unity3d/Team Cherry/Hollow Knight/
```

**Comment by syyePhenomenol** [comment]
Also, did you try turning the "Vanilla Pins" setting in the pause menu on?

**Comment by DIVIDERE** [comment]
I'm having the same issue; I can infinitely buy map markers and can't place any. enabling vanilla markers doesn't fix it and neither does disabling the mod from the pause menu. if I buy markers with the mod removed and then add it again the markers disappear even with vanilla pins enabled.

here's my [mod log](https://pastebin.com/vukH08Jz) and here's my [player log](https://pastebin.com/Rc0hxq5v).

**Comment by syyePhenomenol** [comment]
Hi all,

This issue has been fixed in my local code of MapChanger, but I will need to make some changes with RandoMapMod before everything gets updated on Scarab.

**Comment by syyePhenomenol** [comment]
Should now be fixed with MapChanger v1,0.6 and VanillaMapMod v2.0.2
